packages/sanity/src/core/form/inputs/InvalidValueInput/converters.ts

```diff
@@ -56,6 +57,32 @@ export const converters: {[fromType: string]: {[toType: string]: ValueConverter}
         }
       },
     },
+    portableText: {
+      test: TRUE,
+      convert: (value: string) => [
+        {
+          _type: 'block',
+          _key: randomKey(12),
+          children: [{_type: 'span', text: value, _key: randomKey(12)}],
+          markDefs: [],
+        },
+      ],
+    },
+  },
+  text: {
+    portableText: {
+      test: TRUE,
+      convert: (value: string) => [
+        {
+          _type: 'block',
+          _key: randomKey(12),
+          children: value
+            .split('\n')
+            .map((line) => ({_type: 'span', text: line, _key: randomKey(12)})),
```

Comment:
This isn't quite right, it will convert:

```
This
is

a test
```

to:
```
thisisatest
```

A span is a "chunk of text", in a line. We should _either_ leave the newlines as-is, or try to map it to multiple blocks.

Here's a `convert` function that converts to individual blocks instead (note that it only matches _double_ newlines - single newlines are treated as "soft newlines". We could change to single newlines if preferable.

```ts
convert: (value: string) => value.split('\n\n').map(line => ({
  _type: 'block', // or `blockSchemaType.name`, actually
  _key: randomKey(6),
  children: [{ _type: 'span', _key: randomKey(6), text: line }],
  markDefs: []
}))
```